TestApp/TestApp/CustomViews/AtomicComponents/Molecules/TwoButtonView.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  TwoButtonView.swift
+//  ECCBWallet
+//
+//  Created by Jonathan Bott on 11/4/19.
+//  Copyright © 2019 Jon Bott. All rights reserved.
+//
+
+import Foundation
+import DesignSystem
+
+public class TwoButtonView: UIView {
+
+    private let style1: Button.Style
+    private let style2: Button.Style
+    private var closure1: VoidClosure = { }
+    private var closure2: VoidClosure = { }
+
+    public init(style1: Button.Style, style2: Button.Style, buttonWidth: CGFloat, padding: CGFloat) {
+        self.style1 = style1
+        self.style2 = style2
+        super.init(frame: .zero)
+        guard let buttonBrand = BrandingManager.brand as? ButtonBrand else { Assert("BrandingManager.brand does not conform to ButtonBrand"); return }
+        
+        addSubview(button1)
+        addSubview(button2)
+        
+        let size = Button.Size(width: .half, height: .large)
+        
+        let offset = buttonWidth/2 + padding/2
+        
+        button1.pin(to: self, [.top(.xxsmall), .centerX(-offset),  .width(asConstant: 164), buttonBrand.heightPin(for: size)])
+        button2.pin(to: self, [.top(.xxsmall), .centerX(offset), .width(asConstant: 164), buttonBrand.heightPin(for: size)])
+        // Potentially change this widthPin / heightPin setup
```

Comment:
Little bit of confusion here with the size of the buttons... you're passing `buttonWidth` but not using it. and also not using the `buttonBrand.widthPin` 164 is a magic number... I'm guessing you'd want a setup that gives you padding on either side and in between the two buttons.. for that use

`button1.pin(to: self, [.leading(.keyline), .custom(.trailing, to: .centerX, constant: -.keyline/2)])`
`button2.pin(to: self, [.trailing(-.keyline), .custom(.leading, to: .centerX, constant: .keyline/2)])`

I think the button brand pinning stuff is confusing and bad... potentially it needs removing. It was mainly to make different heights and widths for different brands.